scripts/test-a11y.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+cd `dirname $BASH_SOURCE` && cd ..
+
+if [ -z "$BASE_URL" ]; then
+    echo "Error: BASE_URL must be set to point to your sandbox"
+    exit 1
+fi
+
+mkdir -p test/logs
+cd test/logs
+export SELENIUM_BROWSER=phantomjs
+
+echo "Running accessibility tests against the sandbox..."
+python ../acceptance/accessibility.py
```

Comment:
I was wondering the same. I could also see where someone might want to run both, one right after another.